Project.toml

```
@@ -0,0 +1,21 @@
+name = "XMLDict"
+uuid = "228000da-037f-5747-90a9-8195ccbf91a5"
+version = "0.4.0"
+
+[deps]
+DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
+EzXML = "8f5d6c58-4d21-5cfd-889c-e3ad7ee6a615"
+IterTools = "c8e1da08-722c-5040-9ed9-7db0dc04731e"
+
+[compat]
+DataStructures = "0.11, 0.12, 0.13, 0.14, 0.15, 0.16, 0.17"
+EzXML = "0.7, 0.8, 0.9, 1.0, 1.1"
+IterTools = "1.0, 1.1, 1.2, 1.3"
```

```suggestion
IterTools = "1.0"
```
This will be compatible with [1.0, 2.0)